package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "probe-dictionary-mozilla",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.3.1",
+    "d3-array": "^2.2.0",
+    "d3-axis": "^1.0.12",
+    "d3-scale": "^3.0.0",
+    "d3-selection": "^1.4.0",
+    "d3-shape": "^1.3.5",
+    "env-cmd": "^9.0.3",
+    "jquery-param": "^1.0.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-refetch": "^2.0.3",
+    "react-scripts": "^3.0.1"
```

Comment:
I know it's too late now, but *Next.js* is great alternative to CRA and worth considering in the future.